File: app/src/main/java/com/selepdf/hackovid/model/Pack.java
package com.selepdf.hackovid.model;

public class Pack extends GeneralItem{

    private Product[] products;

    public Pack(String name, String description, String thumbnail, float price, float rating, String[] images, Product[] products) {
        super(name, description, thumbnail, price, rating, images);
        this.products = products;
    }

    public Product[] getProducts() {
        return products;
    }

    public void setProducts(Product[] products) {
        this.products = products;
    }
}

File: hackovid_be/model/Pack.js
var mongoose = require('mongoose');

var PackSchema = new mongoose.Schema({  
  name: String,
  images: [String],
  description: String,
  products: { type: [mongoose.Schema.ObjectId], ref: "Product" },
  price: Number,
  rating: Number,
});
mongoose.model('Pack', PackSchema);

module.exports = mongoose.model('Pack');